flask-app/secrets.py

```diff
@@ -0,0 +1,4 @@
+user = 'admin'
+password = 'password'
+db = 'climactic_test'
```

Comment:
let's not do this security abomination maybe 👁️

Reply:
Oops 👀 my bad